lib/datasource/local/models/amount_view_model.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/model/token_model.dart';
+
+class AmountViewModel {
```

Comment:
actually. Looking deeper into this. This is not a ViewModel. 

ViewModels are the ones we use to Model a view, they hold the state. 

This is just a data model.  
FiatDataModel
CryptoDataModel

Reply:
We could make it a data model for sure.

Having 2 classes is fine, I think, since everywhere they're being used it's clear if it's fiat or crypto. I will make those changes.

Reply:
And I agree 100% that's better than a bool - I just was thinking that it's cool that we don't need to worry if something is fiat or crypto, all we need to know is the amount, the precision, and the symbol. But yeah.

Reply:
Did that too, now FiatDataModel and TokenDataModel, it's a lot better that way, thanks!